### PR TITLE
Armor buff

### PR DESCRIPTION
**Pull request by Sparrowbird611**
## About The Pull Request

Rebalances armor, primarily buffing low to mid tier armors, and buffs laser beam AP.

## Why It's Good For The Game

Hopefully makes armor a little more meaningful. I've personally found that anything less that 40% did very little to help me survive. I believe this will make armor tiers that aren't BP more effective. I also standardized the stats a bit because I suspect I have OCD and the numbers were triggering me, so helmets will line up with their vests.

In case anyone is worried, bulletproof armor did not get any stronger against ballistics, 60% is still the highest you'll see on ship. I strengthed some of the lesser specialist armors without making them completely useless against their weakness (looking at you flak armor), so with energy weapons hopefully becoming more prominent, we shouldn't see too many issues in that department.

Also I disabled ablative's deflection cause it was wack.

I have laser bias. That is all. Jokes aside, lasers would be hit too hard with these changes if I didn't give them AP. Plasma have enough to stand on their own so they weren't boosted.

## Changelog
:cl:
balance: Rebalance armor numbers. In general, lower tier armors should be more valuable now, and some things that didn't seem like they should be so armored were reduced.
balance: Gave lasers to compensate for armor change, plasma already has enough to be sustainable.
balance: Removed ablative's ability to deflect projectiles, to make it less of a hard counter to certain weapons.
tweak: Renamed heavy armor vest to mercenary armor vest.
/:cl:

### CODE REVIEW

**Comment by Sparrowbird611** [comment]
I leave it up to maintainers if they want to merge this in combination with the bullet nerf PR or if they want to let this sit for a bit.

**Comment by ValoTheValo** [comment]
> I leave it up to maintainers if they want to merge this in combination with the bullet nerf PR or if they want to let this sit for a bit.

(please dont)

**Comment by SirRichardFrancis** [comment]
absolutely_ruin

Balance PR №5172: NT nerf and contractor buff. Exactly what we need.

**Comment by SirRichardFrancis** [comment]
image

It was sarcastic.

**Comment by Sparrowbird611** [comment]
> image
> 
> It was sarcastic.

But I was not.

**Comment by ValoTheValo** [comment]
> absolutely_ruin
> 
> Balance PR №5172: NT nerf and contractor buff. Exactly what we need.

absolute state of eris development. another balance pr lets fucking goooooooo

**Comment by SirRichardFrancis** [comment]
Also, on the note of buffing energy weapon...

image

**Comment by Sparrowbird611** [comment]
> 

I already have plans for this, do not fret my child.

**Comment by ValoTheValo** [comment]
paramedic armor is now worse than literal bucket helmet

**Comment by SirRichardFrancis** [comment]
Yeah, literally plastic bucket put on a head is 50% better than super expensive corporate goon armor. Yeah. I love balance PRs.

**Comment by ValoTheValo** [comment]
> Yeah, literally plastic bucket put on a head is 50% better than super expensive corporate goon armor. Yeah. I love balance PRs.

who would win
elite moebius paramedic with top of the line platinum armor
some hobo fuck with a bucket on his head

**Comment by SirRichardFrancis** [comment]
Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items more common, thirdly, one and possibly more items price changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

**Comment by Sparrowbird611** [comment]
Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items more common, thirdly, one and possibly more items price changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

**Comment by ValoTheValo** [comment]
> Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items more common, thirdly, one and possibly more items price changes are direct violation to contributing guidelines, you should try reading them. https://wiki.cev-eris.com/Content_GuidelinesEn

I don't make PRs i am the commenter of PRs and you broke the rules buddy, https://wiki.cev-eris.com/Content_GuidelinesEn you should try reading these. This PR is useless and you should be ashamed you ever made it, get out of my sight.

File: code/modules/clothing/suits/armor.dm
Comment (Humonitarian) [inline code/modules/clothing/suits/armor.dm:580]: I don't see why this nerf is necessary, personally. The crusader armor is already on the lower end due to its raised stiffness.

File: code/modules/clothing/suits/armor.dm
Comment (Humonitarian) [inline code/modules/clothing/suits/armor.dm:245]: Platecarrier shouldn't rock better bomb resistance than armors dedicated against it, such as the flak.

File: code/modules/clothing/suits/armor.dm
Comment (Sparrowbird611) [reply code/modules/clothing/suits/armor.dm:580]: I hate 5%. That is all.

File: code/modules/clothing/suits/armor.dm
Comment (Sparrowbird611) [reply code/modules/clothing/suits/armor.dm:245]: Honestly idk what the difference between a plate carrier and flak vest is, pretty sure I just made them identical stat wise cause they were pretty similar ingame.

File: code/modules/clothing/suits/armor.dm
Comment (Sparrowbird611) [reply code/modules/clothing/suits/armor.dm:580]: I'll put it back.

File: code/modules/clothing/suits/armor.dm
Comment (ValoTheValo) [reply code/modules/clothing/suits/armor.dm:245]: > Honestly idk what the difference between a plate carrier and flak vest is, pretty sure I just made them identical stat wise cause they were pretty similar ingame.

Flak vest costs more in the serb vendor for more protection. Dont make it the fucken same

Comment (Humonitarian) [reply code/modules/clothing/suits/armor.dm:245]: Flak vest was actually underpowered, it had much less bullet protection in exchange for more bomb protection

Comment (Sparrowbird611) [reply code/modules/clothing/suits/armor.dm:245]: Ok so what do you recommend for plate carrier to put it between a flak vest and a regular vest.

Comment (Humonitarian) [reply code/modules/clothing/suits/armor.dm:245]: > Ok so what do you recommend for plate carrier to put it between a flak vest and a regular vest.

Platecarrier ideally should only be better at bullet than flak.

File: code/modules/clothing/suits/armor.dm
Comment (Humonitarian) [reply code/modules/clothing/suits/armor.dm]: You could raise it to 30 too if you want

File: code/modules/clothing/suits/armor.dm
Comment (Sparrowbird611) [reply code/modules/clothing/suits/armor.dm:245]: But I can't get any better at bullet without it just being BP.

Comment (Sparrowbird611) [reply code/modules/clothing/suits/armor.dm]: Resolved, it has far more rounded out stats than the flak now with less bomb protection.

Also just hear me out, I thought a flak vest was meant to protect you from shrapnel, which is covered by bullet resistance. Idc either way though.

File: code/modules/clothing/spacesuits/rig/suits/combat.dm
Comment (Humonitarian) [inline code/modules/clothing/spacesuits/rig/suits/combat.dm:70]: What the absolute uh

Comment (Sparrowbird611) [reply code/modules/clothing/spacesuits/rig/suits/combat.dm:70]: FeAtUrE

**Comment by TheShown911** [comment]
With bullet damage being decreased I would hold off of this one for a little while to see how things feel.

**Comment by Humonitarian** [comment]
Doesn't buff top-tier armor like bulletproof, which are used by everyone, so I don't think this one will make too much difference, especially in the context of style.

**Comment by Firefox13** [comment]
I agree with Shown - let's hold on this for now and see how much #6972 gives us sufferings

**Review by Firefox13 — APPROVED** [review]
https://youtu.be/nyKnKbhAyu0

**Pull request merged**
